apps/styleguide/src/forms/InputOntology.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <FormGroup v-bind="$props" v-on="$listeners">
+    <div class="p-0 m-0">
+      <InputOntologySubtree
+        class="pl-0"
+        :terms="terms"
+        :selection="selection"
+        :expanded="expanded"
+        @toggleExpand="toggleExpand"
+        @select="select"
+        @deselect="deselect"
+      />
+    </div>
+  </FormGroup>
+</template>
+
+<script>
+import _baseInput from "./_baseInput";
+import TableMixin from "../mixins/TableMixin";
+import FormGroup from "./_formGroup";
+import InputOntologySubtree from "./InputOntologySubtree";
+
+/**
+ * Expects a table that has as structure {name, parent{name} and optionally code, definition, ontologyURI}
+ *
+ * Known limitations: this version retrieves complete ontology and renders in place. Purpose is to enable sensible data entry and limited use in filter user interface.
+ *
+ * For future versions we have many ideas for improvements, for example:
+ * - want to make it more lazy, only retrieving the 'root' elements and loading the children when needed.
+ * - adding search to pre-filter elements / paths
+ */
+export default {
+  extends: _baseInput,
```

Comment:
Add name

Reply:
Is that always required? I tend to only add 'name' when the component can be used in a nested way.